config/schema_config.yaml

```diff
@@ -507,6 +536,14 @@ clo_subtype_of:
   source: clo
   target: clo
 
+cl_subtype_of:
```

Comment:
You should define similar specified relationships for `part_of` and `has_part`. Currently, `part_of` and `has_part` relationships are defined between `ontology term` nodes, which is a parent type.